### PR TITLE
add homepage promo carousel

### LINKED ISSUE #10: Promo carousel in front page
Implement Promo carousel 

#### To do : 

- [x] Hugo promos content type (image, title, subtitle, link)
- [x] Front page widget
Screen Shot 2021-01-11 at 12 45 41 PM

https://invis.io/S5ZPUVV9FBQ#/440396978_OCW_NexGen_Homepage_12-11-2020_V4_Ferdi-s_Idea

### PR DESCRIPTION
**Pull request by aliceriot**
#### Pre-Flight checklist

- [ ] Screenshots and design review for any changes that affect layout or styling
  - [ ] Desktop screenshots
  - [ ] Mobile width screenshots
  - [ ] Tag @ferdi or @pdpinch for review
- [ ] Testing
  - [ ] Code is tested
  - [ ] Changes have been manually tested

#### What are the relevant tickets?

closes #10 

#### What's this PR do?

implements the design for the 'promo' carousel. it's based on banner images which are managed via markdown files.

#### How should this be manually tested?

check it out locally and make sure it works alright

#### Screenshots (if appropriate)

Screenshot from 2021-01-15 10-10-09
Screenshot from 2021-01-15 10-10-02

### CODE REVIEW

**Comment by aliceriot** [comment]
updated to follow the new plan:

Screenshot from 2021-01-19 12-33-37
Screenshot from 2021-01-19 12-32-23
Screenshot from 2021-01-19 12-32-15

File: README.md [hunk 1]
```diff
@@ -25,3 +25,19 @@ the filename of `firstname-lastname.md` will allow hugo to automatically pull
...
+
+```
+hugo -s site new promos/filename.md
+```
+
+You'll then also need to check the image file into
+`site/static/images/promo-carousel/` and add it to the front matter in the file
+you just created, alongside the URL for the promo.
+
+You'll also need to set a background color to show behind the image on wide
```
Comment (Ferdi) [inline README.md:42]: No need for background color anymore ?

Comment (aliceriot) [reply README.md:42]: nope

Comment (aliceriot) [reply README.md]: forgot to update the README, should be accurate now

**Review by mbertrand — COMMENTED** [review]
Looks great 👍

**Comment by Ferdi** [comment]
@aliceriot  A couple of comments before I forget . Feel free to open a new PR

* We should remove the top/bottom orange lines
* Add some space on top and right of the image
Screen Shot 2021-01-19 at 1 43 10 PM
* The height of each item should be the same 
* @pdpinch said the podcast logo should be square

/cc @abdulkdawson

**Comment by aliceriot** [comment]
on the logo, it's much easier to do this kind of layout and have it be responsive and so on if we can have the same aspect ratio (at least roughly) for the different images - I asked @abdulkdawson for a rectangular podcast image for that reason. so if we could avoid a square one I think that is definitely easier

I think the heights should be the same - which one was off?

**Comment by Ferdi** [comment]
> I think the heights should be the same - which one was off?

I think the first one is a bit short

**Comment by aliceriot** [comment]
updated:
Screenshot from 2021-01-19 14-24-56

**Review by mbertrand — APPROVED** [review]
👍

**Pull request merged**